src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIOTalonFX.java

```diff
@@ -117,23 +137,27 @@ public void updateInputs(ShooterIOInputs inputs) {
     boolean leftConnected =
         BaseStatusSignal.refreshAll(
                 leftPosition,
-                leftVelocityRadiansPerSecond,
+                leftVelocityRotationsPerSecond,
+                leftVelocitySetpointRadiansPerSecond,
+                leftVelocityErrorRadiansPerSecond,
                 leftAppliedVolts,
                 leftCurrentAmps,
                 leftTemperatureCelsius)
             .isOK();
     boolean rightConnected =
         BaseStatusSignal.refreshAll(
                 rightPosition,
-                rightVelocityRadiansPerSecond,
+                rightVelocityRotationsPerSecond,
+                rightVelocitySetpointRadiansPerSecond,
+                rightVelocityErrorRadiansPerSecond,
                 rightAppliedVolts,
                 rightCurrentAmps,
                 rightTemperatureCelsius)
             .isOK();
     boolean acceleratorConnected =
         BaseStatusSignal.refreshAll(
                 acceleratorPosition,
-                acceleratorVelocityRadiansPerSecond,
+                acceleratorVelocityRotationsPerSecond,
                 acceleratorAppliedVolts,
                 acceleratorCurrentAmps,
                 acceleratorTemperatureCelsius)
```

Comment:
Make sure to set the status signal parameters in the same order that the inputs are created in ShooterIO.java